File: src/main/java/com/telegram/folobot/constants/BotCommandsEnum.java
package com.telegram.folobot.constants;

public enum BotCommandsEnum {
    SILENTSTREAM("/silentstream"),
    FREELANCE("/freelance"),
    NOFAP("/nofap"),
    FOLOPIDOR("/folopidor"),
    FOLOPIDORTOP("/folopidortop");

    public final String label;

    BotCommandsEnum(String label) {
        this.label = label;
    }

    public static BotCommandsEnum valueOfLabel(String label) {
        for (BotCommandsEnum command : values()) {
            if (command.label.equals(label)) {
                return command;
            }
        }
        return null;
    }
}

File: src/main/java/com/telegram/folobot/constants/BotCommandsEnum.kt
package com.telegram.folobot.constants

enum class BotCommandsEnum(val label: String) {
    SILENTSTREAM("/silentstream"),
    FREELANCE("/freelance"),
    NOFAP("/nofap"),
    FOLOPIDOR("/folopidor"),
    FOLOPIDORTOP("/folopidortop");

    companion object {
        fun valueOfLabel(label: String): BotCommandsEnum? {
            for (command in values()) {
                if (command.label == label) {
                    return command
                }
            }
            return null
        }
    }
}